core/arch/arm/pta/crypto_aux.c

```diff
@@ -0,0 +1,75 @@
+/* SPDX-License-Identifier: BSD-2-Clause
```

Comment:
minor: check expected format

Reply:
yes, I though the C source way was `// SPDX-License-Identifier: xxx`

Reply:
@etienne-lms there are a bunch of files with C-style comments, for example. So which one to choose? :)
https://github.com/OP-TEE/optee_os/blob/master/core/include/io.h

Reply:
@igoropaniuk see rules 1.2.1 and 1.2.2 in https://github.com/OP-TEE/optee_os/blob/master/documentation/copyright_and_license_headers.rst.

Reply:
check these guidelines: https://github.com/OP-TEE/optee_os/blob/master/documentation/copyright_and_license_headers.rst#new-source-files.